### PR TITLE
Move data management capabilities to fleet section

### PR DESCRIPTION
**Pull request by npentrel**

### CODE REVIEW

File: docs/how-tos/upload-data.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ aliases:
   - /services/data/upload/
 ---
 
-If you configured [cloud sync](/services/data/capture-sync/), Viam automatically uploads data from the specified directory to the cloud, at the interval you specified.
+If you configured [cloud sync](/services/data/), Viam automatically uploads data from the specified directory to the cloud, at the interval you specified.
```
Comment (sguequierre) [inline docs/how-tos/upload-data.md:21]:
```suggestion
If you configured [cloud sync](/services/data/), Viam automatically uploads data from the specified directory to the cloud at the interval you specified.
```

File: docs/services/data/_index.md [hunk 2]
```diff
@@ -1,147 +1,742 @@
...
 
-The [data client API](/appendix/apis/data-client/) supports the following methods:
+If cloud sync is enabled, the data management service deletes captured data once it has successfully synced to the cloud.
 
-<br>
+With `viam-server`, the data management service will also automatically delete local data in the event your machine's local storage fills up.
+Local data is automatically deleted when _all_ of the following conditions are met:
 
-Methods to upload data like images or sensor readings directly to the Viam cloud:
+- Data capture is enabled on the data manager service
```
Comment (sguequierre) [inline docs/services/data/_index.md:105]:
```suggestion
- Data capture is enabled on the data management service
```

File: docs/services/data/_index.md [hunk 3]
```diff
@@ -1,147 +1,742 @@
...
+
+To capture data from one or more machines, you must first [configure the data management service](#data-management-service-configuration).
+Then [configure data management](#resource-data-capture-configuration) on each {{< glossary_tooltip term_id="resource" text="resources" >}} that you want to capture data from.
+
+### Data management service configuration
+
+{{< tabs >}}
+{{% tab name="Config Builder" %}}
+
+From your machine's **CONFIGURE** in [the Viam app](https://app.viam.com), add the `data management` service.
```
Comment (sguequierre) [inline docs/services/data/_index.md:131]:
```suggestion
From your machine's **CONFIGURE** tab in [the Viam app](https://app.viam.com), add the `data management` service.
```

File: docs/services/data/_index.md [hunk 4]
```diff
@@ -1,147 +1,742 @@
...
+- **Rentention policy**: Set a `retention_policy` attribute in your [data capture configuration](#resource-data-capture-configuration) to avoid keeping data stored in the Viam cloud longer than a specified number of days.
+- **Pausing sync**: You can pause cloud sync at any time by navigating to your machine's **CONFIGURE** tab and disabling **Syncing** for your [data management service](../).
+
+  If you have captured data that you do not want to sync, delete the data on the machine before resuming cloud sync.
+  To delete the data locally, `ssh` into your machine and delete the data in the directory where you capture data.
+
+- **Sync data conditionally**: You can use a {{< glossary_tooltip term_id="module" text="module" >}} to sync data only when a certain logic condition is met, instead of at a regular time interval.
+  For example, if you rely on mobile data but have intermittent WiFi connection in certain locations or at certain times of the day, you may want to trigger sync to only occur when these conditions are met.
+  To set up triggers for syncing see:
```
Comment (sguequierre) [inline docs/services/data/_index.md:712]: was this supposed to say something else or indeed supposed to lead to API? just checking

**Review by sguequierre — CHANGES_REQUESTED** [review]

File: docs/services/data/_index.md (hunk 4 above)
Comment (npentrel) [inline docs/services/data/_index.md:712]:
```suggestion
  To set up triggers for syncing see [Trigger cloud sync conditionally](/how-tos/trigger-sync/).
```

**Review by sguequierre — APPROVED** [review]

**Pull request merged**